apps/snapodds/src/app/pages/odds/pipes/odds-ordering.pipe.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { Sports } from '@response/typings';
+import { OddType } from '../../../models/odd-type';
+import { OddsOrderingPipe } from './odds-ordering.pipe';
+
+describe('OddsOrderingPipe', () => {
+  const pipe = new OddsOrderingPipe();
+
+  it.each`
```

Comment:
I introduced a new library for testing, allowing for spock like testing syntax (jest-each)